index.html

```diff
@@ -9,17 +9,20 @@
   <body>
     <main>
       <section id="intro">
-        <p>TEST1</p>
-      </section>
-      <section id="about">
-        <p>TEST2</p>
-      </section>
-      <section id="projects">
-        <p>TEST3</p>
-      </section>
-      <section id="contact">
-        <p>TEST4</p>
+        <div class="hamburger-menu">
+          <div class="bar1"></div>
+          <div class="bar2"></div>
+          <div class="bar3"></div>
+        </div>
+
+        <div class="header-intro">
+          <h1 class="name">CODE.<span class="square">IT();</span>LAB</h1>
```

Comment:
Ovu css klasu name prebaci u header-intro-title -> ako smo nazvali njen container header-intro, onda pratimo semantički što njegov child radi, njegov child je title, pa inheretno dobiva klasu od parenta header-intro + njegova akcija, odnosno title.